CHANGELOG.md

```diff
@@ -1,13 +1,36 @@
 # App Center SDK for React Native Change Log
 
-## Version 1.10.0 (Under development)
+## Version 1.10.0
 
+### Features
+
+AppCenterAnalytics
+
+[Android] Add API to specify event persistence priority.
```

Comment:
Add `-` to format list items.
`[Android]` => `**[Android]**`

Reply:
Done